evap/staff/templates/staff_evaluation_textanswers_section.html

```diff
@@ -52,6 +53,15 @@
                                         </div>
                                     </form>
                                 </td>
+                                <td class="text-center">
+                                    <form class="full-textanswer-flag-form" method="POST" action="{% url 'staff:evaluation_textanswers_update_flag' %}">
+                                        {% csrf_token %}
+
+                                        <input type="hidden" name="answer_id" value="{{ answer.id }}" />
+
+                                        {% include "staff_evaluation_textanswer_flag_radios.html" with is_initially_flagged=answer.is_flagged id_suffix=answer.id %}
```

Comment:
Maybe an interface consideration: `id_suffix`: The suffix part is kind of an implementation detail, I'd say. For the API, I'd just make it `id` or `textanswer_id` or similar.